apps/shipments/models.py

```diff
@@ -414,6 +418,10 @@ def timestamp_str(self):
     def has_gps(self):
         return True if self.source.lower() == 'gps' else False
 
+    def set_point(self):
```

Comment:
Instead of having a set_point method on the model, please have `point` be populated in a pre_save signal handler for `TrackingData`

Reply:
Since most fields of `TrackingData` are required, I will set this field in serializer